.dotnet/examples/Chat/Example01_SimpleChat.cs

```diff
@@ -11,7 +11,7 @@ public void Example01_SimpleChat()
     {
         ChatClient client = new(model: "gpt-4o", Environment.GetEnvironmentVariable("OPENAI_API_KEY"));
 
-        ChatCompletion completion = client.CompleteChat("Say 'this is a test.'");
+        ChatCompletion completion = client.CompleteChat(new UserChatMessage("Say 'this is a test.'"));
```

Comment:
Just to check, including tagging @KrzysztofCwalina -- we're absolutely sure about this? I know we can add it back later, but when it's written here in the update to "Example 1," it's very clear that it's a hit to the happy path.

Reply:
yeah, why is this being removed?

Reply:
I restored `ChatMessage`'s implicit cast from `string` to `UserChatMessage`.